<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>751</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>调节阈值</string>
   </property>
   <widget class="QSlider" name="horizontalSlider_low1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>240</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_low2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>240</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_low3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>240</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_high1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>240</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_high2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>240</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_high3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>240</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_low1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>55</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_low2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>95</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_low3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>135</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_high1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>55</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_high2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>95</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_high3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>135</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>低阈值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>高阈值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>59</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道一</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>98</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道二
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>通道三</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>59</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道一</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>98</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道二</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>135</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>通道三</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="chooseBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>35</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>violet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>brown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>165</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
